src/main/java/com/example/sccshop/ProductListResponse.java

```diff
@@ -0,0 +1,19 @@
+package com.example.sccshop;
+
+import java.util.List;
+
+/**
+ * @author Eddú Meléndez
+ */
+public class ProductListResponse {
+
+	private List<ProductResponse> products;
+
+	public List<ProductResponse> getProducts() {
+		return this.products;
```

Comment:
We could remove `this` from here as it's superfluous. However, I know that this is the convention mostly used in Spring sources, so if you prefer it, we can leave it as is.

Reply:
I would prefer keeping `this`

Reply:
Ok. Let's keep it then :)